### ISSUE TITLE
Paladin judgements with macros doesnt works

### ISSUE THREAD

**Issue by xergio** [elysium-project/tbc-bug-tracker#675]
**Summary:**

Take this simple macro:

```
#showtooltip
/cast Judgement
/cast Seal of Righteousness
```

**Steps to Reproduce:** 

1. Use `Seal of the Crusader` to place the seal on yourself (or any other seal like wisdom or light)
2. Use the macro to judge a mob.

**Expected Results:** 

The mod should take the `Judgement of the Crusader` debuff, or the debuff of the seal choosen, in addition to place the `Seal of Righteousness` buff on yourself after the debuff is applied.

**Actual Result:**

No debuff applied, and while the animation of the `Judgement` is happening the `Seal of Righteousness` buff is applied, and when the animations ends the `Seal of Righteousness` is consumed.

**Additional Information:**

This is the standart macro used back in days and in all tbc pservers, it works perfectly in servers like Smolderforge.

**Comment by Ziggy-QA** [comment]
@xergio This is not working due to the change that allowed for paladin seal twisting. Currently, you may find macros not working since there is now a delay between the spells cast and the application of its auras. This delay is random at 0-400 ms however consistant. This is something that needs adjustment, or reverted. 

Thanks for your report regardless!

**Comment by xergio** [comment]
Yes, I tried to do it without macro and the same problem happens, if you place a Seal too fast after the Judgement, the Seal is consumed...
